addon/integrations/google-analytics.js

```diff
@@ -59,8 +59,13 @@ export default Base.extend({
     }
 
     for (let key in options) {
-      const capitalizedKey = capitalize(key);
-      gaEvent[`event${capitalizedKey}`] = options[key];
+      const value = options[key];
+
+      // If key is not a 'dimension' or 'metric', prepend with 'event'
+      let shouldPrefix = !/^(dimension|metric)[0-9]{1,2}/.test(key);
```

Comment:
Although I would use `const` over `let` simply due to the fact we know it wont be changed during the runtime and, of course, neither do we want it to be